js/navigation.js

```diff
@@ -65,10 +65,10 @@
 		var self = this;
 
 		// Move up through the ancestors of the current link until we hit .nav-menu.
-		while ( -1 === self.className.indexOf( 'nav-menu' ) ) {
+		while ( self.classList.contains( 'nav-menu' ) ) {
```

Comment:
The same thing here.

```suggestion
		while ( ! self.classList.contains( 'nav-menu' ) ) {
```